docs/manual/src/docs/asciidoc/_includes/servlet/saml2/saml2-login.adoc

```diff
@@ -1610,3 +1610,33 @@ http {
 The success handler will send logout requests to the asserting party.
 
 The request matcher will detect logout requests from the asserting party.
+
+[[servlet-saml2login-store-authn-request]]
+=== Storing the Authentication Request
+
+The `Saml2AuthenticationRequestRepository` is responsible for the persistence of the `AbstractSaml2AuthenticationRequest` from the time the Authentication Request is initiated to the time the Authentication Response is received.
+The `Saml2AuthenticationTokenConverter` is responsible for loading the `AbstractSaml2AuthenticationRequest` from the `Saml2AuthenticationRequestRepository` and saving it into the `Saml2AuthenticationToken`.
+
+The default implementation of `Saml2AuthenticationRequestRepository` is `HttpSessionSaml2AuthenticationRequestRepository`, which stores the `AbstractSaml2AuthenticationRequest` in the `HttpSession`.
+
+If you have a custom implementation of `Saml2AuthenticationRequestRepository`, you may configure it by exposing it as a *Bean* as shown in the following example:
```

Comment:
Instead of *emphasizing* words like "Bean", we should mark them as code, like `@Bean`.